src/Java.md

```diff
@@ -186,6 +186,7 @@ It would be appreciated if there are any Java VTK experts who could convert any
 [ColorAnActor](/Java/Visualization/ColorAnActor) | vtkActor vtkCanvas | Switch style interaction adds C and A key actions for selecting Camera or Actor interaction, and T and J key actions for Trackball or Joystick interaction mode. Adds charEvent observer callback to restore the missing E and Q events to quit.
 [Cursor2D](/Java/Visualization/Cursor2D) | vtkCursor2D | Generates a 2D Cursor Representation.
 [Cursor3D](/Java/Visualization/Cursor3D) | vtkCursor3D | Generates a 3D Cursor Representation
+[CursorShape](/Java/Visualization/CursorShape) | | Changes the shape of Cursor. 
```

Comment:
Change this line: "Changes the shape of Cursor" to: "Change the shape of the cursor."

Reply:
have made changes. please check.